tests/test_fixtures/rule_based_profiler/quentin_user_workflow_verbose_profiler_config.yml

```diff
@@ -46,6 +47,7 @@ rules:
           allow_relative_error: $variables.allow_relative_error
         false_positive_rate: $variables.false_positive_rate
         num_bootstrap_samples: $variables.num_bootstrap_samples
+        bootstrap_random_seed: $variables.bootstrap_random_seed
```

Comment:
@NathanFarmer So basically the inclusion of `bootstrap_random_seed` makes tests predictable obviating the need for the `probabilistic_test` decorator, right?

Reply:
@alexsherstinsky That is correct.